packages/react-csrf/src/hooks.ts

```diff
@@ -5,9 +5,9 @@ import {CsrfTokenContext} from './context';
 export function useCsrfToken() {
   const csrf = useContext(CsrfTokenContext);
 
-  if (csrf == null) {
```

Comment:
This looks like it should have been updated when the type changed from `string | null` to `string | undefined` but was missed because of the fuzzy equality.